examples/run_CPMG.py

```diff
@@ -37,7 +37,7 @@
     ph1_cyc = 0.0
     nPhaseSteps = 1
 # }}}
-# {{{better tau
+# {{{better tau for a perfectly symmetric echo
```

Comment:
better tau was too ambiguous the point here was to generate a symmetric echo by accounting for the delays like deadtime, markers, etc